app/controllers/vacation_requests_controller.rb

```diff
@@ -33,6 +33,7 @@ def create
     if @vacation_request.save && create_approval_request(managers_ids)
       render  status: :created,
               json: @vacation_request
+      UserNotifier.send_confirm_email(current_user).deliver_now
```

Comment:
We should have the same behaviour in `approval_requests_controller`, for `create` action method.